google/resource_compute_instance.go

```diff
@@ -40,7 +40,7 @@ func resourceComputeInstance() *schema.Resource {
 		Update: resourceComputeInstanceUpdate,
 		Delete: resourceComputeInstanceDelete,
 
-		SchemaVersion: 2,
+		SchemaVersion: 4,
```

Comment:
It never got set to 3, but there's a migration fn hanging out that migrates from 2 to 3 so the next unclaimed value is 4.

Reply:
👍 

That sounds terrifying, only because it means there's dead code that we'll be turning on. But that also, that's not this PR's fault, and from what I can see, the code doesn't change that much.

Sounds good to me, let's just do some heavy manual testing prior to cutting the release to make sure we don't end up in a weird place.